google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmServerConfiguration.java

```diff
@@ -33,7 +35,8 @@
   private String cloudProjectName;
   private String googleUserName;
 
-  private EventBus projectNameListener = new EventBus();
+  @Transient
```

Comment:
Yes it does. Its actually documented in their plugin docs: http://www.jetbrains.org/intellij/sdk/docs/basics/persisting_state_of_components.html